Dockerfile

```diff
@@ -3,7 +3,15 @@ WORKDIR /app
 RUN apk add npm
 COPY ./package.json /app
 COPY ./package-lock.json /app
-RUN npm i --no-dev
+# templates are needed in order to compile css
+COPY ./tmpl /app/tmpl
+# unsafe-perm makes sure we can write css to node_modules
+RUN npm i --no-dev --unsafe-perm
+
+FROM composer:1.9 as composer
```

Comment:
1.10 and 2 are already available, please use either of those: https://hub.docker.com/_/composer